packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -84,12 +170,76 @@ export default defineComponent({
 				dropCursor(),
 				EditorView.updateListener.of((viewUpdate: ViewUpdate) => {
 					if (!viewUpdate.docChanged) return;
+
+					highlighter.removeColor(this.editor, this.plaintextSegments);
+					highlighter.addColor(this.editor, this.resolvableSegments);
+
 					this.$emit('valueChanged', this.doc);
 				}),
 			);
 		}
 		const state = EditorState.create({ doc: this.query, extensions });
 		this.editor = new EditorView({ parent: this.$refs.sqlEditor as HTMLDivElement, state });
+
+		highlighter.addColor(this.editor, this.resolvableSegments);
 	},
 });
 </script>
+
+<style module lang="scss">
```

Comment:
Many (all?) of these CSS rules apply specifically to the inline expression editor output, which already has them. Do we need the duplication?

Reply:
No, this was left here by accident. Will remove them.